spec/client_spec.js

```diff
@@ -419,15 +419,15 @@ describe('Client', function() {
         it('resolves when the request succeeds', function(done) {
           triggerEvent(subject, 'request:' + requestsCount + '.done', response);
           promise.then(function() {
-            expect(doneHandler).to.have.been.calledWith(response.responseArgs[0]);
```

Comment:
This is coming from [jQuery's ajax](http://api.jquery.com/jquery.ajax/) [here](https://github.com/zendesk/zendesk_app_framework/blob/4cb0caaa644da761e4f32e2efb8f1eaa4c08833d/lib/request.js#L123-L126), right? I'd expect the first argument to be the payload body, no? I don't think there's an ideal solution, but maybe we can merge all arguments into one object?